nbs/docs/capabilities/08_cross_validation.ipynb

```diff
@@ -0,0 +1,1846 @@
rqnM76gSUtJEJXTLC4neWyHRG9LmamPDzhR/edzyen3JFFYuAmLzFoRX8PMBQoUeC2k1b4RbGmCNIiCIZGG34BWOEj0o1QhyQkWh0okTtBJdHIAUxd+h2PEXD1qJnpVzfyF4JXpGMomuayph+5Apcw671TIyDz5ON0Wda0Q+n+crX/kKX/nKV8jnpyCAVJ3HIsdwX/lgqaReNKQ5ii9BfEuBTThkJVoKOJcWamlpSnRBogslulxP9Jeqj3Nm/hbY8ZS060KF0Aq5nhG/MEUYn8S5IOb2r//1DPmSia77J/zbIgZZwjxZWAAdiySMchJcvtIRH4XhySg7c1fi/R1cwzxzBwbFutcGRVEcS5eR1BTrjkwSEpyuD7RwpRtM1voglLxAIS+PNC0olfVgMuE/47o9BRzLr6D2kSd/lt8e+zt+UTpVqp0LwBvzf4Q7vgLpYSnX5Z5vov3mrbxYexLwn6VSCRe1i1TvuhU+sQ7mHeHrukDlGdAjzp6s1cO7ojNmYGgWISgsJa3rimdNor2PKFqVcowXrHdELXuz6eZ14BaRNomesH3GPYeKistp1n2VZv/mhhPWaa2hUjo/NtwLv30H5+V+BUi0c7Ehridrz8CJn4BPrWfNIRdb15dF+odaGDn0Xfyq9DKp3WBNT/TaUfNdv/TSS/nCF75Q9d8XLVrET3/6UymDcmOfffZxEpDdfz7wgQ8AlsLgsssuY/78+USjUU466SSeemriRjyXy/GhD32Inp4e4vE4Z511Flu2bJE+1iaCg1AFegmHamuQL++eiDHHL7T++7qoK4aqWBvdCRudAPGvZwd4xTV3s2k4TWfM4KQD+qp+rtjIBOEV/HxAvX7oUCExpYZWVYEorElToge4YRCKFL8qotagVLyTkBZe2IbmywKrPdZYOxcvB7bJcEj0oDzRnWBReZveIJ47QeAoClJJMq+FoEKhwOc+9zk+97nPUShM8bvpWMQX2y7nwsIl0goUYJG8WdM+pEpVottKXlM+iV7QLaJXGhFZnKxEl6smyyAC+SSoj9PDcO9V8OCPK3Z1UV2aSkt4yCoyxmpDzO3r71xNoYQUO5fWgAOSTVfnh8wDfMmeuyWJJLrxs1dyb/hilirbPM1dYekykp6CBLbV3dJ8u0sVO5e0sMSQVbTSKyR6WUYnRUkEtlbm62TiacZ1ewrsYidQlkuSEe9hm7GEnbQTlaTyF/uGC5Q/wd3/C8kdUq4r5lXODKEq/om9Coluv8va5kG8GzQJ9+HkS+Edf4UDX+2yA6z/uoqi0Nc6hS/6ga+CDz8Kr/u+/7EKuHylHRK9hjViunm9/JGvcK3xbeLZfnnjdCNvkd1jJeu56/CpRP9P9xv4ZuGNDCldvoe2C+z5m7JJdCl7h/Ht8NQfOLTwBCApWLRctvYO+TRZe+7KFGO4ryctWLR1DluOvYwriudJHas4UzdCDLenQ97OJyA8+OCDbN++3flz2223AfCmN70JgP/93//lW9/6Fv/3f//Hgw8+yNy5cznttNNIJCoVtYsvvpg//OEP/OpXv+Lee+8lmUzyqle9ilKpqVDdU1BRTNf/Qm5rkHpzT0TFE73++xrWNSftvVG+6N+/ax3ZQplj9+3i5g+e4BxupkLM8eduPudeUK8fOsyOnYtf4tQ5sAU4T2QFX7Y0yBPdTyuuG40KFg3EziWgeyw2/H5sfSZDHExkbnrFRt9vIWUyvBaCdF3nggsu4IILLkDXp/49O8FgEkneWEgnI0gimRYeghzBkGqPAlDWYhRNlVJR0nNntw3nbQdIGR0fArGQTkZmkSKxHW6/DP51pbTcjAkIWXseTaKlgK7rvPvd7+aVR8xDV8GQYudiEKLAu4q/xrz185ZtgUSYwtOfCGGJB3jTVnabEj3nxbNWUkMYHnxjHSV6eop76HQmyFKi26ShFna6KKUp0VWNJ198DRfkP85wQYLtl01A5k3r+YpO8a6oZd2eDPEef6G5EvOL3fCDE/2PdRLShYpntwwI0U5WvCtkFdmE77H9nvBbDBRnNkeJLhM9y2GfE6B9obOf9np/hXhnh7uDPNIOXUst0l8WnOctRKZodVTXUoSfbl53bPg7r9buJ5KX1I0wGad9Ec75FasjVlaAXyX6owvO49rS6+lXq4vSPGNSF5uUvU7IKmRHsX53UpToqQH4333hyoVkS9Y8kE2iV/Jf5BUFhapdKokeatw5fk9HzbP55ptvrunzzjrrLM+DmQqTgxu++tWvsmzZMk488URM0+Tqq6/ms5/9LK9//esBuO6665gzZw433ngjF110EWNjY/z4xz/m5z//OaeeeioAN9xwA4sWLeL222/n9NNPlzreJoKBOAx5UaK3NshSYE+EILe8KNEB9u2Js2k4zfqdKY5bKnFjUwXDdjvth09ZPmNYqDh4NJXo3iDu9XSFisloREingKM+9kmcNmLMFVLa30anLeg2ZxtC0eyXfBIFzMCV6AHYuQSh9i+XzYp3rESityUIJbo4BMskIKmsy/WqccLhMD/84Q+n/RzZfvMw2c5FIvHw3nv50u8fZPVTOc6SrES/dc67+cCOM/nc0oM4SMYFX/1tdrz4S/z0m3ejqYq8Nmes35VQopv5FL7LNUJZqIVdXR/y5rAWtg7wWiljqddU//ciHA7z7f/3PV71hZ9ybinBzw88w/c1xXvxo/rv4D7gpR8Hrd33dR3YJHrWlKtEN40W0maYQkmeRaBizwnFZWdSDwRJNbWdi2Ql+hnftKyCWueQzlu2hTKLbKllr+b2f97P0oKENcdF8MLU77Ra1u3JEB17RTSUclHevRV48Eccv/EJblcOJhbaT8oldU0lFtKsMF8FeQVXV7FVxp5hFyX6w9fB9pVw6JtgyfG+ry+QyfsrYM5ps94JgwnJv/vJsOewqVXWnFr2D9PNayXcAinQCpbVqbQuKIH5RwKw5bmngTE64v6U6IHaz9jzN2l3sUnZ94atbLuoaXfzyVCiu0JxxfWkvde2
```

Comment:
I don't see that in the pic. In the pic I only see two cross-validation windows, but no test set window?